includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
By adding another value you introduce inconsistency with https://github.com/Wikia/config/blob/dev/CommonExtensions.php#L3064 . So please either rename the variable in config to more descriptive name or add your variable to config too.